### PR TITLE
[FEAT] Adds `/releases` page

### LINKED ISSUE #63: Feature Request: Release Page
### Is your feature request related to a problem? Please describe.

Displaying releases at one place for all Wasmedge related projects.

### Describe the solution you'd like.

Creating a release page with all the project by fetching Changelog from main repos.

Here is WIP demo for release page: https://render.sahil.world/docs/releases

### Describe alternatives you've considered

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by kokamkarsahil**
## Explanation

This PR adds page at `/docs/releases` and does the following changes:

- Adds  `react-markdown` & `classnames` as dev dependenices  f648463377bc7f057f6f6cd008c0c82f1009ac94
-  Adds `ReleaseNotes` component 0e7884b3e02028511b88e1d615e7617074c8f876
- Adds markdown file for `/releases` which can be modified further along with link to the footer for the same page 8b9e9234c98c9df166e774f6a6b35f33672d0c7b

##  Related issue

Closes #63 

## Preview Changes

https://render.sahil.world/docs/releases

### CODE REVIEW

**Comment by alabulei1** [comment]
Hi  @kokamkarsahil ,

Sorry for my late reply. The release page looks great. Could you fix the conflicts first?

I'm also wondering if we can also add wasmedge-rust crate, WasmEdge GO SDK, and WasmEdge C API here?

**Review by alabulei1 — APPROVED** [review]

**Comment by kokamkarsahil** [comment]
> Sorry for my late reply.

No problem.

>  The release page looks great. Could you fix the conflicts first?

Sure, before fixing conflicts the main branch needs to fix https://github.com/WasmEdge/docs/actions/runs/5041542766/jobs/9041329340#step:7:19 else this branch will also break as a node dependency is causing the build to fail it.
 
> I'm also wondering if we can also add wasmedge-rust crate, WasmEdge GO SDK, and WasmEdge C API here?

Yes we can I was planing on adding those but `wasmedge-rust` and `WasmEdge C API` have unmaintained changelog. 

But it can be added in future with no efforts with the below syntax without adding/changing any extra files.

```javascript
<TabItem value ='WasmEdge' label="WasmEdge" default>
    <p><a href="<Github Release Page>">View releases on GitHub</a></p>
    <ReleaseNotes url='<changelofurl>' />
 </TabItem>
```

I will add `WasmEdge GO SDK` changelog and resolve conflicts once the main branch is fixed.

Thank You!

**Comment by alabulei1** [comment]
I'm not sure why there are conflicts again. Could you fix it again?

**Comment by kokamkarsahil** [comment]
@alabulei1 Merge is fixed

Thanks!

File: src/theme/ReleaseNotes/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      .then((response) => response.text())
+      .then((result) => {
+        result = result
+          .split('## ')
+          .filter((item) => !item.includes('# Changelog'))
+          .map((version) => {
+            var [number, ...contents] = version.split('\n')
+            contents = contents.join('\n')
+            return {
+              number: number.replace('\\[', '').replace(']', ''),
```
[the remark this answers is not in the document]
Comment (kokamkarsahil) [reply src/theme/ReleaseNotes/index.js:21]: This isn't SQL query so it's false positive

**Comment by alabulei1** [comment]
Hi, Sorry for my late reply.

Please fix the DCO test. Thanks.

**Pull request merged**